dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -2247,6 +2882,32 @@
       },
       "x-ms-client-name": "DocumentMultiLabelClassificationEvaluationResult"
     },
+    "TextAnalysisAuthoringDocumentSentimentLabelEvaluationResult": {
```

Comment:
This model should probably include the StringIndexType - even if hardcoded - or at least document what the offset is based on. Can the input text be UTF8 or UTF16, with or without a BOM, etc? StringIndexType is used elsewhere as a hint at the very least.